gr-qtgui/python/qtgui/graphicitem.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+from PyQt5.QtWidgets import QLabel
+from PyQt5.QtGui import QPixmap, QPainter
+from PyQt5.QtCore import Qt as Qtc
+from PyQt5.QtCore import QSize
+
+import os
+import sys
+
+from gnuradio import gr
+import pmt
+
+class GrGraphicItem(gr.sync_block, QLabel):
+    def __init__(self,imageFile,scaleImage=True,fixedSize=False,setWidth=0,setHeight=0):
+        gr.sync_block.__init__(self, name = "GrGraphicsItem", in_sig = None, out_sig = None)
+        QLabel.__init__(self)
+        
+        if not os.path.isfile(imageFile):
+            print("[GrGraphicsItem] ERROR: Unable to find file " + imageFile)
+            sys.exit(1)
+            
+        try:
+            self.pixmap = QPixmap(imageFile)
+            self.originalPixmap = QPixmap(imageFile)
+        except OSError as e:
+            print("[GrGraphicsItem] ERROR: " + e.strerror)
+            sys.exit(1)
+        
+        self.imageFile = imageFile
+        self.scaleImage = scaleImage
+        self.fixedSize = fixedSize
+        self.setWidth = setWidth
+        self.setHeight = setHeight
+        super().setPixmap(self.pixmap)
+        super().setMinimumSize(1, 1)
+        self.overlays = {}
+        
+        self.message_port_register_in(pmt.intern("filename"))
+        self.set_msg_handler(pmt.intern("filename"), self.msgHandler) 
+          
+        self.message_port_register_in(pmt.intern("overlay"))
+        self.set_msg_handler(pmt.intern("overlay"), self.overlayHandler)   
+
+    def overlayHandler(self, msg):
+        try:    
+            overlayitem = pmt.to_python(pmt.car(msg))
+            if overlayitem is None:
+                print("[GrGraphicsItem] ERROR: overlay message contains None in the 'car' portion of the message.  Please pass in a dictionary or list of dictionaries in this portion of the message.  Each dictionary should have the following keys: filename,x,y.  Use x=y=-1 to remove an overlay item.")
```

Comment:
should these be changed to `GraphicsItem` instead of `GrGraphicsItem`? 

Also for this whole section, perhaps it should be using the logging api.